### PR TITLE
fix(linux): Do not restart Fcitx

### PR DESCRIPTION
**Pull request by wengxt**
1. Fcitx does not need to be restarted to load newly installed keyman keyboard
2. Restart fcitx may cause issue on kwin/weston Wayland where input method have to be spawned by compositor process. Manually start fcitx will not have permission to be functional with wayland input method protocol.

### CODE REVIEW

**Comment by keyman-server** [comment]
This pull request is from an external repo and will not automatically be built. The build must still be passed before it can be merged. Ask one of the team members to make a manual build of this PR.

**Comment by mcdurdin** [comment]
I will defer to @ermshiperete on review of this -- he may have more comments on why the reset is there at present for fcitx.

**Review by ermshiperete — APPROVED** [review]
Thanks for your contribution! LGTM

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.227-alpha](https://keyman.com/downloads/releases/alpha/17.0.227)

**Pull request merged**
